data-sources/flutter-web/config.yml

```diff
@@ -0,0 +1,37 @@
+id: flutter-web
+displayName: Flutter Web
+description: |
+  ## Why monitor Flutter Web?
+  Monitoring app performance helps you to take a preventative approach to enhance quality rather than waiting for a crash, issue, or user complaint to occur. The Newrelic Flutter APM tool allows you to track important metrics including Apdex ratings, UI hangs, network timeouts and issues, slow launches, tracking custom traces, and more.
+
+  By prioritizing active issues and marking errors as corrected, you may optimize the process once you have a comprehensive picture of all releases. Find out which version the problem first appeared in, merge duplicates, and assess whether anything may get worse in a later release. Add commit information to send deploy emails instantly and suggest an owner for each application error.
+
+  ## What’s included in this quickstart?
+  New Relic's Browser monitoring agent captured the metrics out of the box:
+
+  - Get alerts when your application suffers critical failures and errors..
+  - Dashboards (Throughput, Error rate, Web transaction Time, Apdex Score, Page views, and  more).
+  - Monitor scripts and functions.
+  - Monitor web transactions.
+  
+  ## Comprehensive monitoring quickstart for Flutter Web.
+  
+  Monitor your Flutter applications, and analyze performance bottlenecks while operating in a live development environment. Visually monitor your web user experience across the world with your application rendition via the dashboard.
+
+  Install and start monitoring your data today! 
+
+icon: logo.png
+install:
+  primary:
```

Comment:
```suggestion
  primary:
   link:
     url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/browser-monitoring-integrations/flutter-web-integration/
```

Reply:
Don't commit this but the primary install should be the link install instead I believe.